docs/sql/udf.md

```diff
@@ -297,6 +299,41 @@ Arc<BTreeMap>`, and `feldera_sqlllib::Array` is defined as `type Array
 = Arc<Vec>`.  Currently `feldera_sqlllib::SqlString` is a thin wrapper
 type around the `ArcStr` type from the `arcstr` crate.
 
+A `ROW` type with N fields represented by a Rust `Tup`N datatype.  A
+user-defined structure type is also represented by a tuple `Tup`N
+type.  These tuple types can be imported from the current crate.
```

Comment:
How will the multi-crate PR affect this?

Reply:
it may not at all, have to try it
the one thing I can think of is adding dependencies to TupN to stubs, and perhaps asking users to declare the tuples in udf.rs?

Reply:
It should be sufficient to re-export them in whatever crate we place UDFs in.

Reply:
I have added a test with multi-crate. Turns out that the udf may need tuples, to the globals crate may need dependencies on some Tuple crates...